### PR TITLE
Supplant links with ARDENT

### PR DESCRIPTION
**Pull request by DevinCrowley**
Swap out old links to Daniel's TensorFlow repo and example with corresponding links to ARDENT.

### CODE REVIEW

File: content/tools/reg.yaml [hunk 1]
```diff
@@ -17,9 +17,9 @@ slides:
   link: "https://neurodata.io/talks/tools19.html"
 
 details:
-  - 'Neurodata''s open-source Python <a href="https://github.com/neurodata/image_lddmm_tensorflow">package</a> that performs affine and deformable (LDDMM) image registration.'
+  - 'Neurodata''s open-source Python <a href="https://github.com/neurodata/ardent">package</a> that performs affine and deformable (LDDMM) image registration.'
```
Comment (falkben) [inline content/tools/reg.yaml:20]: Should be NeuroData to be consistent with rest of site and branding (I know this isn't something you did but we might as well change it)

File: content/tools/reg.yaml [hunk 2]
```diff
@@ -17,9 +17,9 @@ slides:
...
 
-  - 'An example jupyter notebook from A. Branch et al. (2019) can be found <a href="https://nbviewer.jupyter.org/github/dtward/image_lddmm_tensorflow/blob/master/Example_iDISCO_rat_waxholm.ipynb">here</a>.'
+  - 'An example jupyter notebook from A. Branch et al. (2019) can be found <a href="https://github.com/neurodata/ardent/blob/master/demo/demo.ipynb">here</a>.'
```
Comment (falkben) [inline content/tools/reg.yaml:22]: can you replace with nbviewer link to the same notebook?  github often fails to render large notebooks.  also, we should, eventually, replace with a gigantum link, so people could actually run it themselves.

**Review by falkben — CHANGES_REQUESTED** [review]

**Comment by falkben** [comment]
please also update the other references to Daniel's code: 

https://github.com/neurodata/neurodata.io/blob/2bb116e6057a099cabf93f19e948521280514fb7/content/tools/reg.yaml#L6

https://github.com/neurodata/neurodata.io/blob/2bb116e6057a099cabf93f19e948521280514fb7/content/tools/reg.yaml#L9

**Comment by DevinCrowley** [comment]
Made redundant PR that supersedes this one.

**Pull request closed without merging**